lib/clusters.nix

```diff
@@ -27,6 +27,16 @@ let
       specialArgs = { inherit nodeName self; };
     };
 
+  mkAMI = nodeName: modules:
```

Comment:
What is the need/difference between using this and using `./pkgs/ami.nix`? I know that the client rebuild script needs unstable nix to function in Mamba for example, so this would not be enough. I also see below that we are still using ami.nix in the overlay so I'm a bit confused.

Reply:
applied, still works :)